modules/detection/webaudio/WebAudioUtils.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Adapter that creates AudioContext objects depending on the browser.
+ *
+ * @returns {AudioContext} - Return a new AudioContext or null if the browser does not support it.
```

Comment:
Update the JSDoc too. You return undefined, not null.

Reply:
Changed.